src/Umbraco.Cms.Api.Management/Controllers/User/ConfigurationUserController.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Asp.Versioning;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Management.Factories;
+using Umbraco.Cms.Api.Management.ViewModels.User;
+using Umbraco.Cms.Web.Common.Authorization;
+
+namespace Umbraco.Cms.Api.Management.Controllers.User;
+
+[ApiVersion("1.0")]
+[Authorize(Policy = "New" + AuthorizationPolicies.RequireAdminAccess)]
```

Comment:
This endpoint requires the user to be able to edit users.
But as far as I know, its only admins that can, so this is why its RequireAdminAccess 🙈